test/general-tests/starknet-plugin-version/check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+# Check if plugin version returns the correct version
+PLUGIN_VERSION=$(npm view @shardlabs/starknet-hardhat-plugin dist-tags.latest)
```

Comment:
Imagine if a commit is made where the version in package.json is incremented. Wouldn't then this test fail, even though it shouldn't?

Reply:
That's right. I think running the command would suffice. https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/181/commits/19453a442c1abff42ae2b47cfca7f103d04df6c3